package.json

```diff
@@ -31,7 +31,20 @@
     "extend": "^3.0.1",
     "message-box": "^0.1.1",
     "mongo-object": "^0.0.3",
-    "underscore": "^1.8.3"
+    "underscore": "^1.8.3",
+    "deep-extend": "0.4.1",
```

Comment:
I believe we removed this `deep-extend` dependency a while back and don't use it anymore. PR was probably branched before we did that